Exercises/4-count-types.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
-const countTypesInArray = null;
-
+const countTypesInArray = list => {
+  const result = {};
+  for (const item of list) {
+    (typeof item in result) ?
+      (result[typeof item] += 1) :
+      (result[typeof item] = 1);
+  }
```

Comment:
Put repeating expressions to local identifiers, for example: `typeof item` and `result[typeof item]`

Reply:
done